src/EnergyPlus/api/datatransfer.py

```diff
@@ -669,3 +736,19 @@ def current_environment_num(self) -> int:
         :return: The current environment number.
         """
         return self.api.currentEnvironmentNum()
+
+    def actual_time(self) -> RealEP:
```

Comment:
Blah. Return value type hint should be ```int```. Same on the next one. Let me know if you want me to fix that.

Reply:
I'll fix it now, I'm doing some other work here anyway.